### PR TITLE
feat: multiplayer docs

### PR DESCRIPTION
**Pull request by bfelbo**
Big revamp of the docs to focus and explain multiplayer more.

### CODE REVIEW

File: docs/docs/advanced/server-side-logic.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 - **Use of non-deterministic runtime built-ins** such as `Date`, `fetch` etc.
 - **Modules** such as `import/export` and `require` as the SDK requires your logic to be self-contained in a single file. If you want to share logic or split the file into multiple you may use a bundler like rollup to produce a single file from many.
-- **Non-serializable data types** such as `Set`, `Map`, `Promise` etc because the game state needs to be able to be represented as `JSON`.
 - **Regular expressions** because they are stateful.
 - **Use of `eval`** because it's potentially harmful and can be used to bypass above rules.
 
 A notable exception to this list is `Math.random()` which is ensured to be deterministic, see [Randomness](randomness.md) to read more about how this works.
 
-The [Rune CLI](cli.md) will warn you if your game logic uses unsafe code so don't worry!
+The [Rune CLI](publishing/cli.md) will also warn you if your game logic uses unsafe code when uploading so don't worry.
```
Comment (helios1138) [inline docs/docs/advanced/server-side-logic.md]: Not sure if it also makes sense to mention something along the lines of "not all flagged code is unsafe and not all unsafe code automatically flagged"

File: docs/docs/advanced/time-based-logic.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 61
 ---
 
-# Time-Based Logic
+# Time-Based Games
```
Comment (helios1138) [inline docs/docs/advanced/time-based-logic.md]: Rename the file to match?

Also, what about smth like "Using Time in your Games"? "Time-Based Games" sounds like it only refers to games that rely heavily on timers or won't work without them, whereas they could only be a minor addition not affecting gameplay

File: docs/docs/publishing/supported-games.md [hunk 3]
```diff
@@ -23,4 +21,4 @@ There's a lot more games you can build, these are just some examples to give you
 
 ## Adding Support for More Games in Upcoming Version
 
-Rune's multiplayer SDK has some great new updates coming, including fast update loop! This'll add support for classics like Snake and modern real-time IO games.
\ No newline at end of file
+We have some great updates coming up, including a 60 fps update loop! This'll add support for classics like Snake and modern real-time IO games.
```
Comment (helios1138) [inline docs/docs/publishing/supported-games.md:24]: 🐍

File: docs/docs/advanced/server-side-logic.md (hunk 1 above)
Comment (bfelbo) [reply docs/docs/advanced/server-side-logic.md]: Fixed in https://github.com/rune/rune-games-sdk/pull/175/commits/19263232dc0660e0d055db617ae9c01fac439646 👍

**Review by helios1138 — APPROVED** [review]
LGTM 💯

File: docs/docs/advanced/time-based-logic.md (hunk 2 above)
Comment (bfelbo) [reply docs/docs/advanced/time-based-logic.md]: Good idea, changed in https://github.com/rune/rune-games-sdk/pull/175/commits/bb268dc290275a4f7726d1601b2539ee0411b050!

**Pull request merged**